src/player.py

```diff
@@ -0,0 +1,10 @@
+class Player:
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+
+def construct_players():
```

Comment:
In my opinion, the game file should be responsible for creating players and not the player file.

The architecture will be clearer if you have on one side the components for the game (every brick like the board, the players) and on the other side the game builder.